cylc/flow/config_schema.py

```diff
@@ -0,0 +1,660 @@
+            'use login shell': [VDR.V_BOOLEAN, True],
+            'login hosts': [VDR.V_STRING, ''],
+            'batch system': [VDR.V_STRING, ''],
+            'cylc executable': [VDR.V_STRING, 'cylc'],
+            'global init-script': [VDR.V_STRING],
+            'copyable environment variables': [VDR.V_STRING_LIST],
+            'retrieve job logs': [VDR.V_BOOLEAN],
+            'retrieve job logs command': [VDR.V_STRING, 'rsync -a'],
+            'retrieve job logs max size': [VDR.V_STRING],
+            'retrieve job logs retry delays': [VDR.V_INTERVAL_LIST],
+            'task event handler retry delays': [VDR.V_INTERVAL_LIST],
+            'tail command template': [
+                VDR.V_STRING, 'tail -n +1 -F %(filename)s'],
+            'batch systems': {
+                '__MANY__': {
+                    'err tailer': [VDR.V_STRING],
+                    'out tailer': [VDR.V_STRING],
+                    'err viewer': [VDR.V_STRING],
+                    'out viewer': [VDR.V_STRING],
+                    'job name length maximum': [VDR.V_INTEGER],
+                    'execution time limit polling intervals': [
+                        VDR.V_INTERVAL_LIST],
+                },
+            },
+        },
+        '__MANY__': {
+            'run directory': [VDR.V_STRING, ''],
+            'work directory': [VDR.V_STRING, ''],
+            'task communication method': [VDR.V_STRING, ''],
+            'submission polling intervals': [VDR.V_STRING, ''],
+            'execution polling intervals': [VDR.V_STRING, ''],
+            'scp command': [VDR.V_STRING, ''],
+            'ssh command': [VDR.V_STRING, ''],
+            'use login shell': [VDR.V_STRING, ''],
+            'login hosts': [VDR.V_STRING, ''],
+            'batch system': [VDR.V_STRING, ''],
+            'cylc executable': [VDR.V_STRING, ''],
+            'global init-script': [VDR.V_STRING, ''],
+            'copyable environment variables': [VDR.V_STRING, ''],
+            'retrieve job logs': [VDR.V_STRING, ''],
+            'retrieve job logs command': [VDR.V_STRING, ''],
+            'retrieve job logs max size': [VDR.V_STRING, ''],
+            'retrieve job logs retry delays': [VDR.V_STRING, ''],
+            'task event handler retry delays': [VDR.V_STRING, ''],
+            'tail command template': [VDR.V_STRING, ''],
+            'batch systems': {
+                '__MANY__': {
+                    'err tailer': [VDR.V_STRING, ''],
+                    'out tailer': [VDR.V_STRING, ''],
+                    'err viewer': [VDR.V_STRING, ''],
+                    'out viewer': [VDR.V_STRING, ''],
+                    'job name length maximum': [VDR.V_STRING, ''],
+                    'execution time limit polling intervals': [
+                        VDR.V_STRING, ''
+                    ],
+                }
+            }
+        },
+    },
+    'monitor': {
+        'sort order': [VDR.V_STRING, 'definition', 'alphanumeric'],
+    },
```

Comment:
Move to a separate client-only configuration file?

Reply:
MS says no after all